plugins/relocatable/rel_brancher.ml

```diff
@@ -33,48 +32,45 @@ module Rel = struct
   let relocations =
     addr_width >>= fun width ->
     relocations >>= fun rels ->
-    external_symbols >>= fun ext ->
-    Fact.return (of_aseq width rels, of_aseq width ext)
+    external_symbols >>= fun exts ->
+    let to_addr = Addr.of_int64 ~width in
+    Fact.return (of_seq rels ~fkey:to_addr ~fdata:to_addr,
```

Comment:
it would be better to first map both sequences, and the use the normal `of_aseq`